js/src/gallery-block/controls.js

```diff
@@ -155,10 +155,23 @@ const Controls = ( { attributes, setAttributes, colors } ) => {
 						setAttributes( { aspectRatio: value } )
 					}
 				/>
-				<p>{ __( 'Look & Feel', 'cloudinary' ) }</p>
+				<p
+					title={ __(
+						'How to resize or crop images to fit the gallery. Pad adds padding around the image using the specified padding style. Fill crops the image from the center so it fills as much of the available space as possible.',
+						'cloudinary'
+					) }
+				>
+					{ __( 'Resize or Crop Mode', 'cloudinary' ) }
+					<span
+						className="dashicons dashicons-info cld-tooltip"
+						data-tooltip="tooltip_auto_sync"
+					>
+						<span id="tooltip_auto_sync" className="hidden"></span>
+					</span>
+				</p>
 				<p>
 					<ButtonGroup>
-						{ LOOK_AND_FEEL.map( ( type ) => (
+						{ RESIZE_CROP.map( ( type ) => (
```

Comment:
@pereirinha - One thing I learnt from Dugi, Map alteres and returns the altered array, meaning you intend on the array to be changed.
In this case you're just going over each item. So, here the `forEach` would be more appropriate.